src/vue/components/word-usage-examples/word-usage-examples-filters.vue

```diff
@@ -85,20 +85,25 @@ export default {
     authorsList () {
       if (this.$store.state.app.wordUsageExamplesReady && (!this.lastTargetWord || this.lastTargetWord !== this.app.homonym.targetWord)) {
         this.lastTargetWord = this.app.homonym.targetWord
-        this.lastAuthorsList = this.app.wordUsageExamples.wordUsageExamples
-          .filter(wordUsageExampleItem => wordUsageExampleItem.author)
-          .map(wordUsageExampleItem => wordUsageExampleItem.author)
-          .filter((item, pos, self) => self.indexOf(item) == pos)
-          .slice()
-
-        this.lastTextWorksList = this.app.wordUsageExamples.wordUsageExamples
-          .map(wordUsageExampleItem => wordUsageExampleItem.textWork)
-          .filter((item, pos, self) => item && self.indexOf(item) == pos)
-          .slice()
-
-        this.removeDisabledFromTypeFilters()
-        this.typeFilter = 'moreResults'
-        this.setDisabledToType('noFilters')
+        if (!this.app.wordUsageExamples.wordUsageExamples) {
+          this.lastAuthorsList = []
+          this.lastTextWorksList = []
+        } else {
```

Comment:
I have added it to fix https://github.com/alpheios-project/components/issues/403

Reply:
Its all is very subjective, and if you don't agree please do not change anything, but I'm thinking if it will be easier to read if we:
```
// Default values
this.lastAuthorsList = []
this.lastTextWorksList = []
// But if we have some data do something more meaningful
if (this.app.wordUsageExamples.wordUsageExamples) {
// A "real" initialization
}
```
What do you think?

Reply:
`lastAuthorsList` and `lastTextWorksList` are rendered to the select options, so I tried to reduce there changes to reduce re-rendering.
That is why I have added here if/else for that.